metadeploy/api/admin.py

```diff
@@ -159,12 +159,14 @@ class PlanTemplateAdmin(MetadeployTranslatableAdmin):
 @admin.register(Plan)
 class PlanAdmin(MetadeployTranslatableAdmin):
     autocomplete_fields = ("version",)
-    list_filter = ("version__product", "tier", "is_listed")
+    list_filter = ("version__product", "tier", "order_key", "is_listed")
```

Comment:
@jgerigmeyer do we want to filter by `order_key`?

Reply:
@jerivas Good catch; typo 👍